src/webclients/components/transformations/transformations.ts

```diff
@@ -215,16 +212,13 @@ export class PassPipeline extends TransformationListItem {
         item.addClass('transformation');
 
         item.on('click', () => {
-            if (vscode) {
-                if (this.list !== undefined) {
-                    this.list.selectedItem = this;
-                    this.list.generateHtml();
-                }
-                vscode.postMessage({
-                    type: 'sdfv.select_transformation',
-                    transformation: this.json,
-                });
+            if (this.list !== undefined) {
+                this.list.selectedItem = this;
+                this.list.generateHtml();
             }
+            TransofrmationListPanel.getInstance().invoke(
```

Comment:
I'm wondering if we can't just make a static method that calls invoke on the only instance